vllm/engine/async_llm_engine.py

```diff
@@ -304,18 +359,24 @@ async def engine_step(self):
 
         # Put the outputs into the corresponding streams.
         for request_output in request_outputs:
-            self.request_tracker.process_request_output(
+            self._request_tracker.process_request_output(
                 request_output, verbose=self.log_requests)
 
+        return bool(request_outputs)
```

Comment:
Nit: Make the code a bit easier to understand:
```suggestion
        return len(request_outputs) > 0
```